libs/asset-viewer/src/lib/components/asset-search-filter-list/asset-search-filter-list.component.scss

```diff
@@ -0,0 +1,26 @@
+ul {
+  display: flex;
+  gap: 0.5rem;
+  flex-wrap: wrap;
+  margin: 1rem 0 0 0;
+  padding: 0 0 1rem 0;
+  border-bottom: 2px solid #dee2e6;
+  list-style: none;
+}
+
+button {
+  background-color: #dee2e6;
```

Comment:
These colors are stored in variables in the _variables.scss file, so we should probably use them from there